models/src/main/protobuf/Either.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+import "scalapb/scalapb.proto";
+import "google/protobuf/any.proto";
+
+option java_package = "coop.rchain.either";
+
+message EitherAny {
```

Comment:
How do I use this from other client languages? Is there cross-language support for `type_url` in protobuf SDKs?

Reply:
@dckc This message is identical to [any.proto](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto)
It shouldn't be hard in other languages to process this type the same way a Google `Any` is handled. Please look at the grpc code in your favorite client language.

Reply:
ok... I put this on my todo list as https://github.com/rchain-community/rchain-api/issues/60

And I opened https://github.com/rchain-community/rchain-grpc/issues/36 for the python guys from proof.